core/src/commonMain/kotlin/com/sourcepoint/mobile_core/network/SourcepointClient.kt

```diff
@@ -123,6 +124,7 @@ interface SPClient {
         legIntCategories: List<String>
     ): GDPRConsent
 
+    @Throws(SPNetworkError::class, SPUnknownNetworkError::class, CancellationException::class)
```

Comment:
`errorMetrics` should never throw an exception. We should instead run the code in a `try/catch` and simply fail silently.